test/specs/__snapshots__/register-unicon-breeder.e2e.js.snap

```diff
@@ -0,0 +1,128 @@
+            </a>
+          </dd>
+        </div>
+      </dl>
+      <h2 class="govuk-heading-m">Contact details</h2>
+      <dl class="govuk-summary-list">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Name</dt>
+          <dd class="govuk-summary-list__value" id="Name">John Doe</dd>
+          <dd class="govuk-summary-list__actions">
+            <a class="govuk-link" href="/preview/draft/e2e-form/whats-your-name?returnUrl=%2Fpreview%2Fdraft%2Fe2e-form%2Fsummary">Change
+              <span class="govuk-visually-hidden">Name</span>
+            </a>
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Email adress</dt>
+          <dd class="govuk-summary-list__value" id="Emailadress">john.doe@example.co.uk</dd>
+          <dd class="govuk-summary-list__actions">
+            <a class="govuk-link" href="/preview/draft/e2e-form/whats-your-email-address?returnUrl=%2Fpreview%2Fdraft%2Fe2e-form%2Fsummary">Change
+              <span class="govuk-visually-hidden">Email adress</span>
+            </a>
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Phone number</dt>
+          <dd class="govuk-summary-list__value" id="Phonenumber">07779875</dd>
+          <dd class="govuk-summary-list__actions">
+            <a class="govuk-link" href="/preview/draft/e2e-form/whats-your-phone-number?returnUrl=%2Fpreview%2Fdraft%2Fe2e-form%2Fsummary">Change
+              <span class="govuk-visually-hidden">Phone number</span>
+            </a>
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Address</dt>
+          <dd class="govuk-summary-list__value" id="Address">1 High Street, Townsville, TS1 1ST</dd>
+          <dd class="govuk-summary-list__actions">
+            <a class="govuk-link" href="/preview/draft/e2e-form/whats-your-address?returnUrl=%2Fpreview%2Fdraft%2Fe2e-form%2Fsummary">Change
+              <span class="govuk-visually-hidden">Address</span>
+            </a>
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Send certificate to same address</dt>
+          <dd class="govuk-summary-list__value" id="Sendcertificatetosameaddress">Yes</dd>
+          <dd class="govuk-summary-list__actions">
+            <a class="govuk-link" href="/preview/draft/e2e-form/do-you-want-your-unicorn-breeder-certificate-sent-to-this-address?returnUrl=%2Fpreview%2Fdraft%2Fe2e-form%2Fsummary">Change
+              <span class="govuk-visually-hidden">Send certificate to same address</span>
+            </a>
+          </dd>
+        </div>
+      </dl>
+      <form method="post" autocomplete="off">
+        <input type="hidden" name="crumb" value="q0fAM6gHWbqN-4VEdtGmiyIae371I8Z0bZ_ds7UI8yd" />
```

Comment:
Ah sorry @KiranVarma02, can we avoid snapshot tests?

See how the hard-coded hidden input `name="crumb"` has been added?

It's something that will vary for each visit so your tests won't pass for every long

Reply:
Reason I used snapshot testing is, I didn't find any other meaningful test to do on summary page. I thought about adding some id or some selector that I can use for testing but snapshot seems easier way to test. 

I didn't notice line 122, thanks for spotting that. I have updated snapshot to just check data.

It passed on multiple runs, I think its okay now. 

Let me know what you think please !